docs/wiki/deployment/yara.md

```diff
@@ -137,3 +185,18 @@ osquery>
 ```
 
 The above is an example of using an absolute path for `sigfile` combined with `pattern`.
+
+### Inline YARA rules with sigrule
+
+Above, we documented how to query the `yara` table using YARA signatures specified in a local file or retrieved from a
+remote host. YARA rules can also be provided inline with the query, using the `sigrule` constraint/column.
+
+For example:
+
+```sql
+osquery> select * from yara where path = '/etc/passwd' and sigrule = 'rule always_true { condition: true }';
+```
+
+To use `sigrule`, first you must set the `enable_yara_table_extension` flag. Because allowing arbitrary YARA rules also
```

Comment:
We removed `enable_yara_table_extension` flag after consensus that it was not needed. `sigrule` is enabled by default. I will quickly test it.